clmm/theory/func_layer.py

```diff
@@ -457,11 +458,15 @@ def compute_reduced_tangential_shear(
 
     z_src_model : str, optional
         Source redshift model, with the following supported options:
-            `single_plane` (default) - all sources at one redshift (if
-            `z_source` is a float) or known individual source galaxy redshifts
-            (if `z_source` is an array and `r_proj` is a float);
-    validate_input: bool
-        Validade each input argument
+
+            * `single_plane` (default): all sources at one redshift (if `z_source` is a float) \
+                or known individual source galaxy redshifts (if `z_source` is an array and \
+                `r_proj` is a float);
+            * `weighing_the_giants_b`: use the equation (6) in Weighing the Giants - III \
+                (https://arxiv.org/abs/1208.0605) to evaluate tangential reduced shear;
```

Comment:
As we discussed during tag up, may I suggest to call that option `applegate14` to have a consistent name scheme with other possible options that will be implemented later.

Reply:
Done.